### PR TITLE
webpack config complete

### PR DESCRIPTION
**Pull request by Harjap1996**
@pryz25 can you check my webpack config

### CODE REVIEW

**Review by pryz25 — APPROVED** [review]
Looks good. But no need to commit package-lock.

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  output: {
+    path: out,
+    filename: 'bundle.js'
+  },
+  module: {
+    rules: [{
+      test: /\.jsx$/,
+      use:{
+        loader: 'babel-loader',
+        options: {
```
Comment (chinkevin25) [inline webpack.config.js:15]: The spacing looks off here? Can't tell if that's Github rendering it oddly. Are we using a linter?

File: package.json [hunk 2]
```diff
@@ -7,11 +7,14 @@
   "engines": {
     "node": ">=6.13.0"
   },
-  "scripts": {
-  },
+  "scripts": {},
   "dependencies": {
+    "webpack": "^4.29.6"
```
Comment (chinkevin25) [inline package.json:12]: webpack should be a developer dependency

**Review by chinkevin25 — COMMENTED** [review]
Are we using a linter?

**Comment by chinkevin25** [comment]
@pryz25 The Package-lock.json is a file that should actually be committed. It acts like a definitive library history. https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore

I wish we didn't have to commit this either!

**Pull request merged**